### DISCUSSION TITLE
Microsoft account locked because sign in using GitHub student?

### DISCUSSION THREAD

**Discussion by immanueldan2204** [community/community#148168] [GitHub Education] [answered]
### Hi everyone,

Hello!

Previously, I applied GitHub student because I need it for my college task using Azure, and it already approved and I received the benefit. So at that time, I tried to sign in to Azure using my GitHub account that have student benefit. But it turn out that Azure doesn't acknowledge my student status and I can't use Azure. Today my Azure account got locked and I can't contact their support because it need sign in first. Kindly have your help, thank you!

![image](https://github.com/user-attachments/assets/22761c3f-8645-40e3-841c-a8ce8026d0c3)

![image](https://github.com/user-attachments/assets/8fbf47d0-6ae9-4488-ba87-2ee1ed95448d)

**Comment by davevad93** [comment, marked as answer]
Hi @immanueldan2204 ,

This is a known issue that has been solved here #135739 , since you can't contact Azure support you should contact GitHub support by opening a ticket [here](https://support.github.com/contact/education), select "`I am a verified student having trouble redeeming my Global Campus offers`" and explain your problem with Azure verification.

**Reply by immanueldan2204** [reply]
Unfortunately Azure support can't help me at all. It's been a year of problem where phone verification from Indonesia number is always failed when I looked at Azure forum. Yes, they ask me to verify phone number, even thought they say student don't need to do it. But guess what they locked my account and ask for phone number verification. Last chance is I hope GitHub support can help me. I already create the ticket. Thank you @davevad93! 

![Screenshot (12)](https://github.com/user-attachments/assets/68fefbfd-f77a-4699-9296-5b1554b36ec3)
![Screenshot (13)](https://github.com/user-attachments/assets/e830fd41-156b-442a-a0f6-93dd5b136b1c)
![Screenshot (14)](https://github.com/user-attachments/assets/c8778210-08ac-499c-a1f6-3f135f30541d)
![Screenshot (15)](https://github.com/user-attachments/assets/e14ecbce-09c7-46c5-8e44-f81bfe2a0fa2)
![Screenshot (16)](https://github.com/user-attachments/assets/304e9b26-ca7a-46d6-9601-bdfe1995696d)
![Screenshot (17)](https://github.com/user-attachments/assets/1cec8244-d3d9-494a-b746-a996bab9ac84)

**Reply by davevad93** [reply]
You're welcome @immanueldan2204 👍
